backend/README.md

```diff
@@ -167,3 +167,98 @@ You can also directly connect to the MariaDB database server with:
 ```bash
 mysql -h 127.0.0.1 -u root
 ```
+
+## Remote Debug the Driver
+
+These instructions assume you are leveraging the Kind cluster in the
+[Run Locally With a Kind Cluster](#run-locally-with-a-kind-cluster) section.
+
+### Build the Driver Image With Debug Prerequisites
+
+Run the following to create the `backend/Dockerfile.driver-debug` file and build the container image
+tagged as `kfp-driver:debug`. This container image is based on `backend/Dockerfile.driver` but installs Delve,
+builds the binary without compiler optimizations so the binary matches the source code, and copies the
+source code to the destination container for the VS Code debugger. Any changes to the Driver code will require
```

Comment:
> VS Code debugger

seems like most of this is pretty generally applicable, in case you want to reword it to make vs code sound optional here